pkg/registry/resource/resourceclaim/strategy.go

```diff
@@ -181,6 +201,45 @@ func toSelectableFields(claim *resource.ResourceClaim) fields.Set {
 	return fields
 }
 
+// authorizedForAdmin checks if the request is authorized to get admin access to devices
+// based on namespace label
+func authorizedForAdmin(ctx context.Context, newClaim *resource.ResourceClaim, nsClient v1.NamespaceInterface) field.ErrorList {
+	allErrs := field.ErrorList{}
```

Comment:
It looks like this function doesn't return a list of errors with length > 1. Would it make sense to simply return an error?

Reply:
Updated to `var allErrs field.ErrorList` instead to be consistent with other functions in pkg/apis/resource/validation/validation.go 
The effect is the same in that no error is returned.